<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDE</class>
 <widget class="QMainWindow" name="IDE">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>731</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>731</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IDE -  Music Generator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/iconMG.png</normaloff>images/iconMG.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>28</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="lineTextEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>31</width>
      <height>371</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="tabStopWidth">
     <number>15</number>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>700</width>
      <height>371</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>700</width>
      <height>371</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>700</width>
      <height>371</height>
     </size>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhMultiLine|Qt::ImhPreferLatin|Qt::ImhSensitiveData</set>
    </property>
   </widget>
   <widget class="QDockWidget" name="musicW">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>780</width>
      <height>371</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>780</width>
      <height>371</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>780</width>
      <height>371</height>
     </size>
    </property>
    <property name="floating">
     <bool>true</bool>
    </property>
    <property name="windowTitle">
     <string>Music Sheet</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_3">
     <widget class="Line" name="line02">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line03">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line04">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line05">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line01">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="lineSeparador">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>3</width>
        <height>61</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>131</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="horizontalSpacing">
        <number>2</number>
       </property>
       <property name="verticalSpacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="negra">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/negra.png</normaloff>images/negra.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="blanca">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/negra.png</normaloff>images/negra.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="corchea">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/negra.png</normaloff>images/negra.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="corcheaDoble">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/negra.png</normaloff>images/negra.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="semicorchea">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/negra.png</normaloff>images/negra.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="silencio">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>images/negra.png</normaloff>images/negra.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line01_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line05_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line02_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line03_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line04_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line01_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line02_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line04_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line03_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line05_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>601</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>601</width>
        <height>373</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>371</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>601</width>
        <height>373</height>
       </size>
      </property>
     </widget>
     <zorder>graphicsView</zorder>
     <zorder>line02</zorder>
     <zorder>line03</zorder>
     <zorder>line04</zorder>
     <zorder>line05</zorder>
     <zorder>line01</zorder>
     <zorder>lineSeparador</zorder>
     <zorder>formLayoutWidget</zorder>
     <zorder>line01_2</zorder>
     <zorder>line05_2</zorder>
     <zorder>line02_2</zorder>
     <zorder>line03_2</zorder>
     <zorder>line04_2</zorder>
     <zorder>line01_3</zorder>
     <zorder>line02_3</zorder>
     <zorder>line04_3</zorder>
     <zorder>line03_3</zorder>
     <zorder>line05_3</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionNew_File"/>
    <addaction name="actionSave_File"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionMusic_Sheet"/>
   </widget>
   <widget class="QMenu" name="menu_Build">
    <property name="title">
     <string>&amp;Build</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Connection"/>
    <addaction name="separator"/>
    <addaction name="actionView_Console"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Build"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCompile"/>
   <addaction name="separator"/>
   <addaction name="actionrun"/>
  </widget>
  <widget class="QDockWidget" name="consoleW">
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>110</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>731</width>
     <height>125</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>console</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <widget class="QPlainTextEdit" name="consoleTextW">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>730</width>
       <height>145</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="plainText">
      <string>&gt;&gt; </string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse</set>
     </property>
     <property name="backgroundVisible">
      <bool>false</bool>
     </property>
     <property name="centerOnScroll">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionCompile">
   <property name="icon">
    <iconset>
     <normaloff>icon/martillo.png</normaloff>icon/martillo.png</iconset>
   </property>
   <property name="text">
    <string>Compile</string>
   </property>
   <property name="toolTip">
    <string>Compilar</string>
   </property>
  </action>
  <action name="actionrun">
   <property name="icon">
    <iconset>
     <normaloff>icon/run.png</normaloff>icon/run.png</iconset>
   </property>
   <property name="text">
    <string>run</string>
   </property>
   <property name="toolTip">
    <string>Run</string>
   </property>
  </action>
  <action name="action_Connection">
   <property name="text">
    <string>&amp;Connection</string>
   </property>
  </action>
  <action name="actionView_Console">
   <property name="text">
    <string>View Console</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionNew_File">
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionMusic_Sheet">
   <property name="text">
    <string>Music Sheet</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="text">
    <string>Save File</string>
   </property>
  </action>
  <zorder>consoleW</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineTextEdit</tabstop>
  <tabstop>textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
